samples/shopify/README.md

```diff
@@ -1,53 +1,71 @@
-# Contentful Shopify Product UI extension
-Add a custom UI field to Contentful that allows users to search and select Shopify products.
-
-![figure](./Shopify-Product-Demo.gif "Contentful Shopify Product UI Extension demo")
-
-
-## Installation
-
-```sh
-git clone git@github.com:contentful/extensions.git
-cd extensions/samples/shopify
-npm install
+# Contentful UI Extension to load products from Shopify's Storefront API
+
+Adds a custom UI field to Contentful that allows users to search and select Shopify products.
+
+![figure](http://contentful.github.io/extensions/assets/Shopify-Product-Demo.gif "Contentful Shopify Product UI Extension demo")
+
+To make the UI extension work you'll need a valid [Storefront API token](https://help.shopify.com/api/reference/access/storefrontaccesstoken) with `unauthenticated_read_product_listings` scope and the API endpoint of your Shopify store. By default the UI-extension uses instance parameters which are pre-configured for demo purposes:
+
+See `extension.json` for the details:
+
+```json
+{
+  "name": "Shopify Products",
+  "id": "contentful-ui-shopify",
+  "fieldTypes": ["Object"],
+  "srcdoc": "./index.html",
+  "parameters": {
+    "instance": [
+      {
+        "id": "apiToken",
+        "name": "Shopify Access Token",
+        "type": "Symbol",
+        "default": "b5392a6d4ba10c47cb4ad1645bf78eb4"
+      },
+      {
+        "id": "apiEndpoint",
+        "name": "Shopify GraphQL endpoint",
+        "type": "Symbol",
+        "default": "https://contentful-demo.myshopify.com/api/graphql"
+      }
+    ]
+  }
+}
 ```
 
-### Configure
+![figure](http://contentful.github.io/extensions/assets//shopify-parameters.jpg "Contentful Shopify UI Extension parameters")
 
-Create a configuration file with your credentials for Contentful.
+## Assign to a field of a content type
 
-```sh
-cp env.example .env
-```
+![figure](http://contentful.github.io/extensions/assets//shopify-object.png "Contentful Shopify Product UI Extension json object")
 
-Open `.env` in a editor of your liking and add your Contentful space ID, and management token. [Learn how to obtain a token](https://www.contentful.com/developers/docs/references/authentication/#getting-an-oauth-token).
+## Usage
 
-Load environment variables
+Make sure you have the Contentful CLI installed:
 
-```sh
-source .env
+```bash
+npm install contentful-cli -g
```

Comment:
It's a bummer that we have to recommend global installation of the cli... as far as I see there is no way to hand in the cma-token via parameter anymore? :( @Khaledgarbaya

Reply:
you should be able to pass the accessToken for all command but it is not the case for the extensions command, @stefanjudis can you create an issue there

Reply:
Done. https://github.com/contentful/contentful-cli/issues/41